tests/api/test_item_types_rest.py

```diff
@@ -45,7 +48,7 @@ def test_item_types_permissions(client, item_type_standard_martigny,
         data={},
         headers=json_header
     )
-    assert res.status_code == 401
+    assert res.status_code == 400
```

Comment:
bad status here, as nobody is logged in we should have a 401. Needs some check. Once it is solved please check similar cases.

Reply:
fixed